jonprl-mode.el

```diff
@@ -48,6 +48,22 @@
   :type 'file
   :group 'jonprl)
 
+(defcustom jonprl-pretty-symbols
+  '(("*" . ?×)
+   ("->" . ?→)
+   ("<>" . ?★)
+   ("=>" . ?⇒)
+   ("=def=" . ?≝)
+   ("base" . ?𝔻)
```

Comment:
For the record, I understood the symbol when I saw it, and thought it was pleasant—I do not think this notation is used by the Nuprl folks, but I (personally) think it is all right. But I don't feel super strongly...